contributing.md

```diff
@@ -0,0 +1,70 @@
+-   Community announcements and asking the community for feedback
+-   Discussing a new release
+-   Asking questions, Q&A that isn't for sure a bug report
+
+### Contributing through Code
+In order to contribute through code follow the steps below. Note that you don't need to be the best programmer to contribute. Our discord is open for questions
+
+ 1. **Pick a feature** you would like to add or a bug you would like to fix
+	 - If you wish to contribute but what you want to fix/add is not already covered in an existing issue, please open a new issue
+ 2. **Discuss** the issue with the rest of the community
+	 - Before you write any code, it is recommended that you discuss your intention to write the code on the issue you are attempting to edit
+	 -  This helps to stop you from wasting your time duplicating the work of others that maybe working on the same issue; at the same time
+	 - This step also allows you to get helpful pointers on the community on some problems they may have encountered on similar issues
+ 3. **Fork** the repository
+	 - A fork creates a copy of the code on your Github, so you can work on it separately from everyone else.
+	 - You can learn more about forking [here](https://docs.github.com/en/get-started/quickstart/fork-a-repo)
+ 4. Ensure that you have **commit signing** enabled
+	 - This ensures that the code you submit was committed by you and not someone else who claims to be you
+	 - You can learn more about how to setup commit signing [here](https://www.freecodecamp.org/news/what-is-commit-signing-in-git/ "https://www.freecodecamp.org/news/what-is-commit-signing-in-git/")
+	 - If you have already made some commits that you wish to put in a pull request without signing them, then you can follow [this guide](https://dev.to/jmarhee/signing-existing-commits-with-gpg-5b58) on how to fix that.
+ 5. **Clone** the repository to your local computer.
+	 - This puts a copy of your fork on your computer so you can edit it
+	 - You can learn more about cloning repositories [here](https://docs.github.com/en/repositories/creating-and-managing-repositories/cloning-a-repository)
+ 6. **Build** the project
+	 - For a detailed look on how to build {{project-name}} look at our [README file](<./README>)
+ 7. **Start writing** your code
+	 - Open up your favorite code editor and make the changes that you wanted to make to the repository
+	 - Make sure to test your code with the test command found in our [README file](<./README>)
+ 8. **Write tests** for your code
+	 - If you are adding a new feature you should write tests that ensure that if someone make changes to the code it cannot break your new feature without breaking the test
+	 - If your code adds a new feature you should also write at least one documentation test. The documentation test's purpose is to demonstrate and document how to use the API feature
```

Comment:
```suggestion
	 - If your code adds a new feature you should also write at least one documentation test. The documentation test's purpose is to demonstrate and document how to use the API feature.
```

Because there are two sentences in the bullet point, both should have a period at the end.